src/pretix/presale/templates/pretixpresale/base.html

```diff
@@ -44,6 +44,9 @@
 </head>
 <body class="nojs" data-locale="{{ request.LANGUAGE_CODE }}" data-now="{% now "U.u" %}" data-datetimeformat="{{ js_datetime_format }}" data-timeformat="{{ js_time_format }}" data-dateformat="{{ js_date_format }}" data-datetimelocale="{{ js_locale }}" data-currency="{{ request.event.currency }}">
 {{ html_page_header|safe }}
+<nav id="skip-to-main" role="navigation" aria-label="{% trans "Skip-Link" %}" class="sr-only on-focus-visible">
```

Comment:
```suggestion
<nav id="skip-to-main" role="navigation" aria-label="{% trans "Skip link" %}" class="sr-only on-focus-visible">
```

Reply:
Hm, you are right, wikipedia doesn’t use a hyphen. For me though „Skip link“ reads like a call to action to skip the link, whereas „skip-link“ (maybe all lowercase?) is the name of a link, that skips something. When thinking of translation, in german I would translate „Skip link“ with „Link überspringen“, whereas „skip-link“ would either stay „skip-link“ (sorry, denglisch) or „Überspringen-Link“. So we would somehow make sure, what is meant here. Should we add a context for translation? Something like `context "a11y-skip-to-nav"`?